storybooknative/src/Button/index.js

```diff
@@ -9,7 +9,7 @@ import { TYPE_OPTIONS, SIZE_OPTIONS, TOKENS } from "./consts";
 
 import type { Props } from "./index";
 
-const getSizeToken = (name, { theme, size }) => {
+export const getSizeToken = (name: string, { theme, size }) => {
```

Comment:
Do you need the `export`?

Reply:
And why is `name` of type `string`? Shouldn't it be more precise? Like just the possibilities from `TOKENS.heightButton`, or something like this.